RecoEgamma/EgammaElectronProducers/plugins/ElectronSeedConverter.cc

```diff
@@ -0,0 +1,105 @@
+#include <iostream>
+#include <string>
+
+#include "FWCore/Framework/interface/global/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/EventSetup.h"
+#include "FWCore/Framework/interface/ConsumesCollector.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "FWCore/ParameterSet/interface/ParameterSetDescription.h"
+#include "FWCore/ParameterSet/interface/ConfigurationDescriptions.h"
+#include "FWCore/Utilities/interface/EDGetToken.h"
+#include "FWCore/Utilities/interface/EDPutToken.h"
+
+#include "DataFormats/TrajectorySeed/interface/TrajectorySeedCollection.h"
+#include "DataFormats/EgammaReco/interface/EleSeedHostCollection.h"
+#include "DataFormats/EgammaReco/interface/SuperClusterFwd.h"
+#include "DataFormats/EgammaReco/interface/SuperCluster.h"
+#include "DataFormats/EgammaReco/interface/ElectronSeed.h"
+#include "DataFormats/EgammaReco/interface/ElectronSeedFwd.h"
+
+
+class ElectronSeedConverter : public edm::global::EDProducer<> {
+public:
+  explicit ElectronSeedConverter(const edm::ParameterSet&);
+
+  void produce(edm::StreamID, edm::Event&, const edm::EventSetup&) const final;
+
+  static void fillDescriptions(edm::ConfigurationDescriptions& descriptions);
+
+private:
+    const edm::EDGetTokenT<TrajectorySeedCollection> initialSeedsToken_;
+    const edm::EDGetTokenT<reco::EleSeedHostCollection> matchedEleSeedSoAToken_; 
+    const edm::EDPutTokenT<reco::ElectronSeedCollection> putToken_;
+	edm::EDGetTokenT<std::vector<reco::SuperClusterRef>> superClustersTokens_;
```

Comment:
The `EDGetToken` could be `const` too, after which I'd suggest to reorder
```suggestion
	const edm::EDGetTokenT<std::vector<reco::SuperClusterRef>> superClustersTokens_;
    const edm::EDPutTokenT<reco::ElectronSeedCollection> putToken_;
```